### PR TITLE
Use braces instead of colon for route elements.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: en/development/routing.rst [hunk 1]
```diff
@@ -592,7 +565,7 @@ you to define name prefixes in each scope::
 You can also use the ``_namePrefix`` option inside nested scopes and it works as
 you'd expect::
 
-    Router::plugin('Contacts', ['_namePrefix' => 'contacts:'], function ($routes) {
+    Router:{plugin}('Contacts', ['_namePrefix' => 'contacts:'], function ($routes) {
```
Comment (markstory) [inline en/development/routing.rst]: I don't think this should have changed.

File: en/development/routing.rst [hunk 2]
```diff
@@ -775,7 +745,7 @@ with the following router connection::
 
     use Cake\Routing\Route\DashedRoute;
 
-    Router::plugin('ToDo', ['path' => 'to-do'], function ($routes) {
+    Router:{plugin}('ToDo', ['path' => 'to-do'], function ($routes) {
```
Comment (markstory) [inline en/development/routing.rst]: Here too.

File: en/development/routing.rst [hunk 3]
```diff
@@ -1577,5 +1531,5 @@ named parameter, will be removed from the list of passed arguments.
     /development/dispatch-filters
 
 .. meta::
-    :title lang=en: Routing
+    {title} lang=en: Routing
```
Comment (markstory) [inline en/development/routing.rst]: Here too.

File: en/development/routing.rst (hunk 1 above)
Comment (ADmad) [reply en/development/routing.rst]: Bad search replaces :slightly_smiling_face:

**Pull request merged**